WalletWasabi.Fluent/ViewModels/Wallets/LoadingViewModel.cs

```diff
@@ -37,7 +40,7 @@ public LoadingViewModel(Wallet wallet)
 
 	private uint RemainingFiltersToDownload => (uint)Services.BitcoinStore.SmartHeaderChain.HashesLeft;
 
-	protected override void OnActivated(CompositeDisposable disposables)
+	protected override void OnNavigatedTo(bool isInHistory, CompositeDisposable disposables)
```

Comment:
This is not good, some of the subscriptions from here should be listened to always, and not just when the LoadingViewModel is being displayed.

Note the difference, on master `OnActivated()` is called once, when the wallet passed the login screen, and doesn't matter what is being displayed on the MainScreen, the LoadingViewModel lives and works always in the background.
With the current solution, it will only detect when the backend is connected (or the fallback mechanism if the backend is down) if the page is currently being displayed and the subscriptions here are alive.

Reply:
@soosr latest commit has a local workaround, which we should remove by the time #10647 gets merged.